lexis/render/histogram.fbs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016, Human Brain Project
+//                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
+
+namespace lexis.render;
+
+table Histogram
```

Comment:
Document better what this type is used for. Just looking at the members it's difficult to guess (what's incarnationId?, why a viewport area?...).
It looks like this histogram is only valid for 8bit dataset, confirm that in the documentation.